### PR TITLE
 spelling/grammar

### PR DESCRIPTION
**Pull request by dplotnikov**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/2607

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbHybrid/hybrid/plan-cloud-auto-attendant.md [hunk 1]
```diff
@@ -40,18 +40,18 @@ Also see:
 
 The following requirements assume that you already have Skype for Business Server 2019 deployed in a supported topology.  Your requirements depend on your scenario:
 
-- If you are already using Exchange UM online or on premises and you upgrade to Skype for Business 2019, you will need to capture the structure of your Auto attendants and re-create them in the cloud using Cloud auto attendants. For more information, see [Moving an Exchange UM auto attendant or call queue to Phone System](configure-onprem-ra.md#moving-an-exchange-um-auto-attendant-or-call-queue-to-phone-system).
+- If you are already using Exchange UM Online or On-Premises and you upgrade to Skype for Business 2019, you will need to capture the structure of your Auto attendants and re-create them in the cloud using Cloud auto attendants. For more information, see [Moving an Exchange UM auto attendant or call queue to Phone System](configure-onprem-ra.md#moving-an-exchange-um-auto-attendant-or-call-queue-to-phone-system).
```
Comment (nenonix) [inline Skype/SfbHybrid/hybrid/plan-cloud-auto-attendant.md]:
```suggestion
- If you are already using Exchange UM online or on-premises and you upgrade to Skype for Business 2019, you will need to capture the structure of your Auto attendants and re-create them in the cloud using Cloud auto attendants. For more information, see [Moving an Exchange UM auto attendant or call queue to Phone System](configure-onprem-ra.md#moving-an-exchange-um-auto-attendant-or-call-queue-to-phone-system).
```

File: Skype/SfbHybrid/hybrid/plan-cloud-auto-attendant.md [hunk 2]
```diff
@@ -40,18 +40,18 @@ Also see:
...
 
 - For a new configuration of Cloud auto attendants, follow the steps outlined in  [Configure resource accounts](configure-onprem-ra.md).
 
 In addition to the requirements above, the below requirements must be configured to connect to the Microsoft Cloud auto attendant service:
 
-- Hybrid connectivity. If you already have Skype for Business Server deployed, and you want to enable Cloud auto attendant for your on-premises users, you must ensure that you have hybrid connectivity set up between your on-premises and online environments. This is sometimes called a split domain configuration.
+- Hybrid connectivity. If you already have Skype for Business Server deployed, and you want to enable Cloud auto attendant for your On-Premises users, you must ensure that you have hybrid connectivity set up between your On-Premises and Online environments. This is sometimes called a split domain configuration.
```
Comment (nenonix) [inline Skype/SfbHybrid/hybrid/plan-cloud-auto-attendant.md]:
```suggestion
- Hybrid connectivity. If you already have Skype for Business Server deployed, and you want to enable Cloud auto attendant for your on-premises users, you must ensure that you have hybrid connectivity set up between your on-premises and online environments. This is sometimes called a split domain configuration.
```

File: Skype/SfbHybrid/hybrid/plan-cloud-auto-attendant.md [hunk 3]
```diff
@@ -77,7 +77,7 @@ Currently we don't support automated migration to the Cloud of a UM auto attenda
 
 1. Use Exchange admin powershell commands to review the structure of the old auto attendant system, including any nested auto attendants and call queues.  
 2. Create copies of text-to-speech scripts or recorded messages associated with each UM auto attendant node.
-3. Create on premise endpoints for each auto attendant node, including assigning a test phone numbers and licenses to the objects. Note that you now have the ability to assign on-premise phone numbers licenses used by online services like Phone System.
+3. Create On-Premises endpoints for each auto attendant node, including assigning a test phone numbers and licenses to the objects. Note that you now have the ability to assign On-Premises phone numbers licenses used by online services like Phone System.
```
Comment (nenonix) [inline Skype/SfbHybrid/hybrid/plan-cloud-auto-attendant.md]:
```suggestion
3. Create on-premises endpoints for each auto attendant node, including assigning a test phone numbers and licenses to the objects. Note that you now have the ability to assign on-premises phone numbers licenses used by online services like Phone System.
```

File: Skype/SfbHybrid/hybrid/plan-cloud-auto-attendant.md [hunk 4]
```diff
@@ -40,18 +40,18 @@ Also see:
...
+- Hybrid connectivity. If you already have Skype for Business Server deployed, and you want to enable Cloud auto attendant for your On-Premises users, you must ensure that you have hybrid connectivity set up between your On-Premises and Online environments. This is sometimes called a split domain configuration.
 
    For more information, see [Plan hybrid connectivity between Skype for Business Server and Office 365](plan-hybrid-connectivity.md) and [Configure hybrid connectivity between Skype for Business Server and Office 365](configure-hybrid-connectivity.md).
 
 - If you're assigning a phone number to to your auto attendant, you will need an [Office 365 Enterprise E5](/skypeforbusiness/skype-for-business-and-microsoft-teams-add-on-licensing/license-options-based-on-your-plan/office-365-enterprise-e5-with-audio-conferencing) license.
-- Create an on-premises [resource account](/MicrosoftTeams/manage-resource-accounts.md) for each auto attendant, and assign phone numbers and licenses. 
+- Create an On-Premises [resource account](/MicrosoftTeams/manage-resource-accounts.md) for each auto attendant, and assign phone numbers and licenses.
```
Comment (nenonix) [inline Skype/SfbHybrid/hybrid/plan-cloud-auto-attendant.md]:
```suggestion
- Create an on-premises [resource account](/MicrosoftTeams/manage-resource-accounts.md) for each auto attendant, and assign phone numbers and licenses. 
```

**Comment by jambirk** [comment]
fixed elsewhere

**Pull request closed without merging**